### ISSUE TITLE
check: missing dependency

### ISSUE THREAD

**Issue by vhf** [mbanting/meteor-reactive-ibeacons#2]
```
I20150924-15:51:57.204(2) (android:http://meteor.local/packages/mbanting_reactive-ibeacons.js:50) Uncaught ReferenceError: check is not defined
```

**Comment by vhf** [comment]
I confirm this patch solves the issue. Without it, `mbanting:reactive-ibeacons` does not run on Android devices.

**Comment by mbanting** [comment]
Thanks very much!

**Comment by mbanting** [comment]
Tagged and published 1.0.3
https://atmospherejs.com/mbanting/reactive-ibeacons
